health_check/management/commands/healthcheck.py

```diff
@@ -0,0 +1,16 @@
+__author__ = 'philroche'
```

Comment:
__author__ is extraneous and the author seems to be following PEP8 recommendation to group imports into 3 groups. http://legacy.python.org/dev/peps/pep-0008/#imports